library/core/src/slice/mod.rs

```diff
@@ -2579,13 +2579,12 @@ impl<T> [T] {
     /// # Examples
     ///
     /// ```
-    /// #![feature(slice_fill)]
-    ///
     /// let mut buf = vec![0; 10];
     /// buf.fill(1);
     /// assert_eq!(buf, vec![1; 10]);
     /// ```
-    #[unstable(feature = "slice_fill", issue = "70758")]
+    #[doc(alias = "memset")]
```

Comment:
Should this be in another pull request?

Reply:
I was thinking of this as a docs patch, which I don't think is uncommon for stabilization PRs. But happy to make this two PRs if folks feel that is more appropriate.

Reply:
> I was thinking of this as a docs patch, which I don't think is uncommon for stabilization PRs.

Indeed. There's not really any reason to split this. Please leave it as is. :)

Reply:
There's https://github.com/rust-lang/rust/pull/79229, which does exactly this btw.

Reply:
Indeed, I missed this one here because the PR description didn't mention anything about adding that too :) Feel free to close my PR or merge it before this one, I don't mind either way.

Reply:
The other PR was merged so this part needs to be updated to fix the conflicts.